hack4impact-utk/compassion-ministries
# 270 display sent emails

# Description
Changed zod type of sentDate field of emails from string to date.
Created EmailList component and added to Event component to display sent emails.
Moved "Send Email" UI from EventView into the Event component.

## Relevant issue(s)

#270 

## Questions


## Type of change


# Checklist:


## to test

Go to the Event page for any Event in the database (make sure it has an `emails` field with an array of objects matching the zod typedef). If `emails` is empty, it should render a disabled accordion, and if not, it should render a populated accordion with the details for each email object.
